azure-sql/database/performance-guidance.md

```diff
@@ -261,7 +261,15 @@ Connection pooling can improve throughput, reduce latency, and enhance the overa
    - [JDBC connection pooling](/sql/connect/jdbc/using-connection-pooling?view=azuresqldb-current&preserve-view=true)
    - [PHP connection pooling](/sql/connect/php/connection-pooling-microsoft-drivers-for-php-for-sql-server?view=azuresqldb-current&preserve-view=true)
 
-- Cloud applications should implement [retry logic](develop-overview.md#resiliency) to handle transient connectivity failures gracefully. Learn more about how to design [retry logic for transient errors](troubleshoot-common-connectivity-issues.md#retry-logic-for-transient-errors). 
+- Cloud applications should implement [retry logic](develop-overview.md#resiliency) to handle transient connectivity failures gracefully. Learn more about how to design [retry logic for transient errors](troubleshoot-common-connectivity-issues.md#retry-logic-for-transient-errors).
+
+- Token-based authentication mechanisms, such as Microsoft Entra ID authentication, may affect connection pooling due to token expiration, which can invalidate connections and interrupt reuse. To mitigate these issues, ensure tokens are refreshed proactively, configure token lifetimes to align with expected connection durations, monitor for expiration errors to trigger reauthentication, and minimize connection churn through effective pooling configurations.
```

Comment:
```
To mitigate these issues, ensure tokens are refreshed proactively, configure token lifetimes to align with expected connection durations, monitor for expiration errors to trigger reauthentication, and minimize connection churn through effective pooling configurations.
```

This may all be good broad advice but we need to be more direct in Docs. Some of these verb phrases are vague or generic. Before we move further in review, please either remove or provide more specifics, including link to other Microsoft Docs, for steps like "ensure tokens are refreshed", "configure token lifetimes".